examples/complex_fluids/ex0/Makefile.am

```diff
@@ -0,0 +1,30 @@
+## Process this file with automake to produce Makefile.in
+include $(top_srcdir)/config/Make-rules
+
+##Dimension-dependent testers
+EXAMPLE_DRIVER = ex_main.cpp
+GTEST_DRIVER   = test_main.cpp
```

Comment:
All the gtest stuff is gone now, so please get rid of this and the other gtest targets/variables.

Reply:
These were the merge conflicts. They are now gone